apps/solid-example/src/routes/atom-demo.tsx

```
@@ -0,0 +1,205 @@
+
+function Counter() {
+  const [count, setCount] = useAtom(counterAtom)
+  return (
+    <div class="p-6 bg-slate-800/50 backdrop-blur-sm border border-slate-700 rounded-xl">
+      <h3 class="text-xl font-semibold text-white mb-4">Counter (useAtom)</h3>
+      <p class="text-3xl font-bold text-cyan-400 mb-6">{count()}</p>
+      <div class="flex gap-4">
+        <button
+          class="px-4 py-2 bg-cyan-500 hover:bg-cyan-600 text-white font-semibold rounded-lg transition-colors"
+          onClick={() => setCount((c) => c + 1)}
+        >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)